packages/react-native-editor/src/test/index.test.js

```diff
@@ -27,158 +29,202 @@ const initGutenberg = ( registerParams ) => {
 	);
 	registerGutenberg( registerParams );
 
-	return render( <EditorComponent /> );
+	let screen;
+	// This guarantees that setup module is imported on every test, as it's imported upon Editor component rendering.
+	jest.isolateModules( () => {
+		screen = render( <EditorComponent { ...editorProps } /> );
+	} );
```

Comment:
With this approach, we no longer need to reset modules for testing the call order of hooks and callbacks. Besides, I found out that resetting modules was breaking the test cases that render the editor.